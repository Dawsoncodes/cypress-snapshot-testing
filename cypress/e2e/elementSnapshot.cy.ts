describe("Element snapshot tests", () => {
  beforeEach(() => {
    cy.visit("https://www.lambdatest.com/selenium-playground/simple-form-demo")
  })

  it("Should match the snapshot of the element", () => {
    // First snapshot
    cy.get("h1").toMatchSnapshot()

    // Second snapshot (Uncomment to see the test fail)
    // cy.get("h2")
    //   .first()
    //   .toMatchSnapshot()
  })
})
