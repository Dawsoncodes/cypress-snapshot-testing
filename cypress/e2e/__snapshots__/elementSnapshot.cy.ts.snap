exports[`Element snapshot tests > Should match the snapshot of the element #0`] = `
<h1
  class="text-size-48 font-bold text-black text-center leading-none text-shadow md:w-full leading-height-70 mx-auto smtablet:text-size-30 smtablet:leading-height-42"
>
  Simple Form Demo
</h1>
`;
